roles/openshift_logging/tasks/oc_apply.yaml

```diff
@@ -18,6 +18,26 @@
   failed_when: "'error' in generation_apply.stderr"
   changed_when: no
```

Comment:
Doesn't seem right to be setting `changed_when: no` on an action that would be mutating the state.